templates/install.tmpl

```diff
@@ -8,6 +8,10 @@
 			<div class="ui attached segment">
 				{{template "base/alert" .}}
 
+				{{/* Drop-down menus do not currently work well without JavaScript, only there to prevent "bad first impressions". */}}
+				{{/* TODO: Remove this together with "enable_javascript" string when that is fixed. */}}
```

Comment:
The "install" page is impossible to work without JS. There are a lot of JS code for the "install" page. See "install.js"

Reply:
Yeah, I found out myself, which is why I tried to add an extra, super loud warning specifically for the install page.

Thanks for the review, I'll try again later.